index.html

```diff
@@ -1,75 +1,110 @@
+      <section class="top-content">
+        <div class="header-content">
+          <nav>
+            <div class="logo">
+              <img src="img/lambda-black.png" />
             </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+            <div class="links">
+              <a href="#Home">Home</a>
+              <a href="about.html">About</a>
+              <a href="#Products">Products</a>
+              <a href="#Blog">Blog</a>
+              <a href="#Contact">Contact</a>
             </div>
-        
-        </section>
+          </nav>
+        </div>
+        <div class="header-img">
+          <img src="img/jumbo.jpg" />
+        </div>
 
-        <section class="bottom-content">
+        <div class="text-container">
+          <h2>The Future</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>The Past</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
+
+        <div class="boxes">
```

Comment:
Another method to style these boxes is to use multiple class names such as class="box box1" or an id class="box" id="box1"